narwhals/expression.py

```diff
@@ -454,9 +454,77 @@ def max(self) -> Expr:
         return self.__class__(lambda plx: self._call(plx).max())
 
     def n_unique(self) -> Expr:
+        """
+         Returns count of unique values
+
+        Examples:
+            >>> import polars as pl
+            >>> import pandas as pd
+            >>> import narwhals as nw
+            >>> df_pd = pd.DataFrame({'a': [1, 2, 3, 4, 5], 'b': [1, 2, 3, 4, 5]})
+            >>> df_pl = pl.DataFrame({'a': [1, 2, 3, 4, 5], 'b': [1, 2, 3, 4, 5]})
```

Comment:
let's include some duplicate values, and maybe have it so that the number of unique values differs for the two columns?

Reply:
ok, can you do that on your end, or you want me to edit it from my side?

Reply:
you can do it from your side, if you make the edit, add, commit, and push, then this pull request will be updated